ruby/neo4j/driver/internal/reactive/abstract_rx_query_runner.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Neo4j::Driver::Internal::Reactive
+  module AbstractRxQueryRunner
+    def parse_query(query, opts)
+      return if query.is_a?(Neo4j::Driver::Query)
```

Comment:
let's don't implement method overloading.

Reply:
hmm ok, shall I remove this module `AbstractRxQueryRunner`?

Reply:
I think so.